web/packages/teleterm/src/ui/Search/pickers/results.story.tsx

```diff
@@ -397,6 +398,11 @@ const AuxiliaryItems = () => (
             ),
           ]}
         />
+        <SuggestionsError
```

Comment:
I moved the story from `ActionPicker.story.tsx` to just `results.story.tsx`. We could have separate stories for `ActionPicker` and `ParameterPicker` but I'm not sure if it makes sense at the moment, we just have a single item from `ParameterPicker` here. I like being able to see them all at once.

Reply:
It's fine to me.